src/tlo/methods/healthsystem.py

```diff
@@ -337,6 +343,45 @@ def make_appt_footprint(self, dict_of_appts):
             "values"
         )
 
+    def get_equip_item_code_from_item_name(self, lookup_df: pd.DataFrame, equip_item_name: str) -> int:
+        """Helper function to provide the equip_item_code (an int) when provided with the equip_item_name of the item"""
+        return int(pd.unique(lookup_df.loc[lookup_df["Equip_Item"] == equip_item_name, "Equip_Code"])[0])
+
+    def set_equipment_essential_to_run_event(self, set_of_equip):
```

Comment:
```suggestion
    def set_equipment_essential_to_run_event(self, set_of_equip: Set[str]) -> None:
```

Not essential but additing a type hint like this makes it explicit what the expected type of the argument is (and that the method doesn't return anything). Would also need to add a `from typing import Set` statement.